nextflow_schema.json

```diff
@@ -0,0 +1,128 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "https://github.com/JAlvarezJarreta/movelib-workflow/main/nextflow_schema.json",
+    "title": "MEvoLib pipeline parameters",
+    "type": "object",
+    "definitions": {
+        "fetch_options": {
+            "title": "Sequence fetching options",
+            "type": "object",
+            "properties": {
+                "query": {
+                    "type": "string",
+                    "description": "NCBI Entrez query."
+                },
+                "species": {
+                    "type": "string",
+                    "description": "Species to fetch sequences for."
+                },
+                "seq_type": {
+                    "type": "string",
+                    "description": "Sequence type."
+                },
+                "refseq": {
+                    "type": "boolean",
+                    "description": "Fetch only reference sequences.",
+                    "default": false
+                }
+            }
+        },
+        "cluster_options": {
+            "title": "Clustering options",
+            "type": "object",
+            "properties": {
+                "cluster_tool": {
+                    "type": "string",
+                    "description": "Clustering tool.",
+                    "enum": ["genes"]
+                }
+            }
+        },
+        "align_options": {
+            "title": "Multiple sequence alignment (MSA) options",
+            "type": "object",
+            "properties": {
+                "align_tool": {
+                    "type": "string",
+                    "description": "MSA tool.",
+                    "enum": ["mafft", "clustalo", "muscle"]
+                },
+                "align_out_format": {
+                    "type": "string",
+                    "description": "Output MSA file format.",
+                    "enum": ["clustal", "emboss", "fasta", "maf", "msf", "phylip", "nexus", "stockholm"],
```

Comment:
For this and inference ones, why are only these formats? 
During ui development, we talked about using the tuple dictionary BioPython provides, but I cannot see all of the possibilities. 
Please, check that variable value by executing MEvoLib GUI and tell me which ones should we keep, as they must be synced up...

Reply:
Again, some laziness from my side. Will check the contents of that field and add them as we agreed to do for the GUI :+1: